codecov.yml

```
@@ -0,0 +1,22 @@
+        flags: 
+          - forms
+        paths: 
+          - packages/forms/
+
+flags:
+  react:
+    paths:
+      - packages/react/
+  forms:
+    paths:
+      - packages/forms/
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)